addons/l10n_il/i18n/l10n_il.pot

```diff
@@ -809,12 +926,53 @@ msgid "Withholding Vendor Tax Account"
 msgstr ""
 
 #. module: l10n_il
-#: model_terms:res.company,invoice_terms_html:l10n_il.demo_company_il
-msgid "Your conditions..."
-msgstr ""
-
-#. module: l10n_il
+#: model:account.account,name:l10n_il.1_il_account_300270
 #: model:account.account,name:l10n_il.2_il_account_300270
 #: model:account.account.template,name:l10n_il.il_account_300270
 msgid "current year earnings"
 msgstr ""
+
+#. module: l10n_il
+#: model:account.tax.report.line,tag_name:l10n_il.account_tax_report_line_out_base
+msgid "הכנסות חייבות במע\"מ"
```

Comment:
Not sure where this comes from as the code in standard does not have the Hebrew originally.  (it should be in the po normally)